composer.json

```diff
@@ -26,7 +26,8 @@
     },
     "require-dev": {
         "mikey179/vfsStream": "~1.5.0",
-        "symfony/var-dumper": "~3.1.3"
+        "symfony/var-dumper": "~3.1.3",
+        "phpunit/phpunit": "^5.4"
```

Comment:
I already merged it. This decision was taken years ago and nothing has changed since then in composer world.